app/stylesheet/ddf_override.scss

```diff
@@ -170,3 +170,39 @@
     padding-bottom: 12px;
   }
 }
+
```

Comment:
If this is just related to button group, add it in a new file.

Reply:
moved the button-group related css to a new file named **button-group.scss** and imported in **application-webpack.scss**